tests/setup_module_utils.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -eu
 # SPDX-License-Identifier: MIT
 
 set -euo pipefail
```

Comment:
Then I guess you should change this to `set -o pipefail` since the `eu` are already in the bash invocation